src/sd.jl

```diff
@@ -415,3 +415,141 @@ function build_constraint(
         shape,
     )
 end
+
+"""
+    HermitianPSDCone
+
+Hermitian positive semidefinite cone object that can be used to create a
+hermitian positive semidefinite square matrix in the [`@variable`](@ref)
```

Comment:
```suggestion
Hermitian positive semidefinite square matrix in the [`@variable`](@ref)
```

I think all usages of `hermitian` should be capitalized?

Reply:
Indeed, you are right